pandora-client-web/src/assets/graphicsLoader.ts

```diff
@@ -1,8 +1,11 @@
 import { Assert, GetLogger, Logger } from 'pandora-common';
-import { BaseTexture, IImageResourceOptions, Resource, Texture, autoDetectResource } from 'pixi.js';
+import { BaseTexture, IImageResourceOptions, Resource, Texture, autoDetectResource, settings as PIXISettings } from 'pixi.js';
 import { PersistentToast } from '../persistentToast';
 import { IGraphicsLoader } from './graphicsManager';
 
+// Prevent PIXI from doing PIXI things
+PIXISettings.STRICT_TEXTURE_CACHE = true;
```

Comment:
hmm could we put this in a file on its own, I remember we had that batch-size thing we no longer need and that was kind of hard to find, so maybe a ts file with a good name could help